daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -494,8 +494,10 @@ func (d *Driver) Remove(id string) error {
 	if err := os.RemoveAll(dir); err != nil && !os.IsNotExist(err) {
 		return err
 	}
-	if err := subvolRescanQuota(d.home); err != nil {
-		return err
+	if quotaEnabled {
```

Comment:
Yes, this will be a problem on restore, for sure.
We need a better way to determine if quota is enabled.